archivist_samples/software_bill_of_materials/README.md

```diff
@@ -18,24 +18,35 @@ Install the [RKVST Python package](https://pypi.org/project/jitsuin-archivist/ "
 Get an authorization bearer token and store it in the file `.auth_token`. If you don't know how to do this, please refer to the [RKVST documentation](https://docs.jitsuin.com/docs/setup-and-administration/getting-access-tokens-using-client-secret/ "Getting an auth token")
 
 
-## Running the sample
+## Running the samples
 
-The sample script creates a SoftwarePackage object and uploads a Software Bill of Materials based on the [standard SWID example](https://www.ntia.gov/files/ntia/publications/ntia_sbom_formats_and_standards_whitepaper_-_version_20191025.pdf "ACME Roadrunner Detector") "ACME Roadrunner Detector".
+### Simple sample
 
-To run it, simply: 
+The simple sample script creates a SoftwarePackage object and uploads a Software Bill of Materials based on the [standard SWID example](https://www.ntia.gov/files/ntia/publications/ntia_sbom_formats_and_standards_whitepaper_-_version_20191025.pdf "ACME Roadrunner Detector") "ACME Roadrunner Detector".
+
+To run it: 
 
 ```bash
 python3 ./sbom_demo.py
 ```
 
 This will create a new Asset in your RKVST tenancy and publish a single update to the SBOM,registering full details for version 4.1.5 with the platform.
 
+### Complex sample
+
+The complex sample takes the concept of assured data one step further and extends to assured processes. It simulates a series of product lifecycle events that result in a variety of updates to the SBOM, including private patches for specific customers and internal planning and approval for major updates.
+
+To run it: 
+
+```bash
+python3 ./sbom_advanced_demo.py
```

Comment:
In archivist-samples we generate an entry point in setup.py and execute that as described in the front-of-house README.md the assumption being that the user will install this a a python packge from pypi.

Only in DEVELOPERS.md do we execute the code as python...

]

Reply:
Fixed in wheel version (update 5)